packages/noco-docs/docs/070.fields/040.field-types/060.formula/070.formula-result-formatting.md

```diff
@@ -0,0 +1,73 @@
+---
+title: 'Formatting Formula Results'
+description: 'This article explains how to format the output of formulas in NocoDB, covering various formatting options based on the result type.'
+tags: ['Fields', 'Field types', 'Formula', 'Formatting']
+keywords: ['Fields', 'Field types', 'Formula', 'Formatting', 'Create formula field', 'Formatting options']
+---
+
+Formatting the output of formulas allows you to tailor how your data is displayed based on the type of result. Various formatting options are available to ensure that your data is presented in the most useful and visually appealing way.
+
+![image](/img/v2/fields/types/formula-format.png)
+
+
+## Numeric Formats
+
+<img src="/img/v2/fields/numeric-format.png" alt="image" width="400"/>
+
+### Decimal
+The `Decimal format` is used to display results with a specified number of decimal places, allowing you to define the precision.
+
+### Currency
+The `Currency format` is ideal for displaying monetary values, with the option to set and display a specific currency symbol.
+
+### Percent
+The `Percent format` displays results as a percentage with the % symbol.
```

Comment:
Percent field currently is without `%` symbol. so, display wise it is currently more like decimal - but with an add on option to configure display as progress